tests/template_tests/test_templatetags.py

```diff
@@ -0,0 +1,16 @@
+from django.templatetags.static import StaticNode
+from django.test import SimpleTestCase
+
+
+class StaticNodesTest(SimpleTestCase):
```

Comment:
I'm sure there must be existing tests for `StaticNode` in another file where these can be moved?

Reply:
I searched through the Django codebase and I didn't find any

Reply:
```suggestion
class StaticNodeTests(SimpleTestCase):
```

Please can you move this test case into `tests/template_tests/syntax_tests/test_static.py`.